docx/blkcntnr.py

```diff
@@ -66,6 +66,23 @@ def tables(self):
         from .table import Table
         return [Table(tbl, self) for tbl in self._element.tbl_lst]
 
+    @property
+    def story(self):
+        """
+        A list containing paragraphs and tables in document order
+        """
+        from .table import Table
+        from docx.oxml.text.paragraph import CT_P
+        from docx.oxml.table import CT_Tbl
+
+        def make_element(el, s):
+            if isinstance(el, CT_P):
+                return Paragraph(el, self)
```

Comment:
Did you mean for this to refer to "self" where line 82 refers to "s"

Reply:
Does it matter? `make_element` is called once with `(.., self)` and no other usage to this function is possible. So it may be either `s` or `self`

Reply:
Does it matter? `make_element` is called once with `(.., self)` and no other usage to this function is possible. So it may be either `s` or `self`

Reply:
@belegnar  Why putting a  `s` argument then? Could you not just use the `self` from the call to `story` method?